### DISCUSSION TITLE
Share file

### DISCUSSION THREAD

**Discussion by fusengine** [minio/minio#12827] [Q&A] [answered]
why when i want to share a file it gets the ip address of docker and not the url, moreover i can't find anything on documentation concerning the change of the API link.

Do you have an idea??

<img width="1070" alt="Capture d’écran 2021-07-29 à 11 13 49" src="https://user-images.githubusercontent.com/7295655/127465963-5f23b5f1-7603-4064-8102-337cf3db58e7.png">

**Comment by mieseprem** [comment, marked as answer]
Have you tried to set MINIO_SERVER_URL?

**Comment by harshavardhana** [comment]
> Have you tried to set MINIO_SERVER_URL?

This one ^^

**Reply by fusengine** [reply]
for me this is not work

```docker
version: '3.9'
services:
  minio:
    restart: always
    image: minio/minio:latest
    environment:
      - MINIO_ROOT_USER=user
      - MINIO_ROOT_PASSWORD=password
      - MINIO_BROWSER_REDIRECT_URL=https://my.domain
      - MINIO_REGION_NAME=eu-central-1
      - MINIO_DOMAIN=https://my.domain
    command:  server /data --console-address ":9001" 
    ports:
      - 9000:9000
      - 9001:9001
    volumes:
      - minio-data:/data

volumes:
  minio-data:for this is not work
````

**Reply by harshavardhana** [reply]
You are missing `MINIO_SERVER_URL`  @fusengine

**Reply by fusengine** [reply]
> You are missing `MINIO_SERVER_URL` @fusengine

Don't Work ==>. {"code":401,"message":"invalid Login"}

I use external reverse proxy and why he dont Work?

But old version Good work and New version have lot problem and the documentation is extremely incompleted.

```sh

original error -> (user_login.go:50: invalid Login)

debugging error: Post "https://s3.mydomain.com/": x509: certificate is not valid for any names, but wanted to match s3.mydomain.com
``

**Reply by harshavardhana** [reply]
> ```shell
> 
> debugging error: Post "https://s3.mydomain.com/": x509: certificate is not valid for any names, but wanted to match s3.mydomain.com
> ``
> ```

Please fix your TLS certs.

> But old version Good work and New version have lot problem and the documentation is extremely incomplete.

It's documented right here https://github.com/minio/minio#test-using-minio-console - if documentation is lacking please send a PR to update.  This is free software please send fixes - happy to accept changes.
